chalicelib_cgap/check_setup.json

```diff
@@ -587,5 +541,100 @@
         "display": [
             "<env-name>"
         ]
+    },
+    "access_key_status": {
+        "title": "Admin Access Key Status",
+        "group": "Maintenance Checks",
```

Comment:
To make life easier, could we create a `maintence_checks.py` module and move the corresponding checks we want under that header there? It doesn't matter for the sake of things working, but it will help with consistency (at least I think the other checks mostly follow this pattern) and ease of look-up of the corresponding code.

Reply:
These checks are common for all ENCODED style systems and thus exist in `foursight-core` ie: https://github.com/4dn-dcic/foursight-core/blob/master/foursight_core/checks/access_key_expiration_detection.py

Reply:
Ah, that's what I was missing. I don't know if we were able to do this previously, but it makes a lot of sense to have these shared. That resolves multiple comments of mine.

Reply:
Yes with the ES7 changes we can write checks both in `foursight-core` and in the sub-repos and have them be picked up. Just cuts down on any duplicate implementations which a lot of these would have if we want them in both.